src/main/java/com/microsoft/jenkins/containeragents/aci/AciCloud.java

```diff
@@ -270,5 +296,15 @@ public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Item owner) {
         public ListBoxModel doFillResourceGroupItems(@QueryParameter String credentialsId) throws IOException {
             return AzureContainerUtils.listResourceGroupItems(credentialsId);
         }
+
+        public ListBoxModel doFillLogAnalyticsCredentialsIdItems(@AncestorInPath Item owner) {
```

Comment:
Missing permission check: https://github.com/jenkinsci/credentials-plugin/blob/master/docs/consumer.adoc#providing-a-ui-form-element-to-let-a-user-select-credentials

example: https://github.com/timja/azure-artifact-manager-plugin/blob/154095c109a8084f3b48aea213a278815843d74a/src/main/java/com/microsoft/jenkins/artifactmanager/AzureArtifactConfig.java#L89-L111

🚫 blocker